### DISCUSSION TITLE
Support immediate regeneration of Pages with revalidate property at `next start`

### DISCUSSION THREAD

**Discussion by scook12** [vercel/next.js#25039] [Ideas]
### Describe the feature you'd like to request

When starting a production build of a next app, user should be able to force a single, immediate regeneration of pages that are using ISR. It's useful for cases where the app itself is built and tagged (i.e. in a docker image or an archive of the app) but the prerendered pages that use dynamic data with ISR are way out of date.

Current approach (as far as I can tell) requires forcing a build of the entire app immediately prior to `next start` or starting the app with stale data for the `revalidate` period of time until ISR naturally kicks in.

### Describe the solution you'd like

When running `next start`, have an option to force an incremental regeneration of pages with the `revalidate` property in their `getStaticProps` implementation. Something like `next start --force-revalidate`.

### Describe alternatives you've considered

You could offer a property like `revalidateAtStart: true` on `getStaticProps` that lets you select ISR pages that will be regenerated at start time as opposed to a CLI option, which would be more granular in choosing which pages would be affected, but that would require code changes in the app to specify (what seems to me) a build process parameter.

**Comment by stephanbakker** [comment]
Could be very helpful in our setup as well, with multiple app instances, using potentially outdated images (content).
First case: Deploying an app that was built in the pipeline a couple of days ago.
Second case: Handling a restart (automatic usually), which could spin up an image that was built a long time ago

Other solution might be an option to not create the pages statically on build, and let ISR kick in at runtime.
